client/src/components/requester_task/partial-confirm.html

```diff
@@ -1,9 +1,11 @@
 <div id="requester-task-confirm">
-  <h1> Please verify before Confirming Order </h1>
-  Location: <span> {{ ctrl.order.vendor }} </span> <br/>
-  Item: <span> {{ ctrl.item }}</span> <br/>
-  Additional Requests: <span> {{ ctrl.details }} </span> <br/>
-  Meeting time: <span> {{ ctrl.time | date : 'h:mm a' }}</span> <br/>
-  Price: <span> {{ ctrl.price | currency }} </span> <br/>
-  <button ng-click="ctrl.confirmOrder()"> Confirm Order </button>
+  <div class="container">
+    <h1 class="requester-task-stage"> Confirm Order </h1>
+    <h4 class="requester-choice"> <small> Location: </small> {{ ctrl.order.vendor }} </h4>

```

Comment:
When you have a list, use the `<li>` with `<ul>` or `<ol>`. You can remove the bullet points created by `<ul>` by CSS setting `list-style: none;`